chatbot/db.py

```diff
@@ -14,7 +14,7 @@
 # TODO: we might want to have a list of available commands somewhere in the class so that we can
 # quickly check before updating so that we don't crash.
 class DbConnector:
-    def __init__(self, db_path: str = "../db/prod/"):
+    def __init__(self, db_path: str = os.path.join(os.path.dirname(__file__), "../db/prod/")):
```

Comment:
yeah we did that on strream already but since the PR for the badges is still unfinished I'm welcoming this change early.